go.mod

```diff
@@ -231,7 +231,7 @@ require (
 
 replace github.com/minio/minio v0.0.0-20210206053228-97fe57bba92c => github.com/juicedata/minio v0.0.0-20220613143934-cee0571a1b03
 
-replace github.com/hanwen/go-fuse/v2 v2.1.1-0.20210611132105-24a1dfe6b4f8 => github.com/juicedata/go-fuse/v2 v2.1.1-0.20210926080226-cfe1ec802a7f
+replace github.com/hanwen/go-fuse/v2 v2.1.1-0.20210611132105-24a1dfe6b4f8 => github.com/qingyunha/go-fuse/v2 v2.1.1-0.20220710020838-753b234a08c0
```

Comment:
please send a PR to github.com/juicedata/go-fuse/v2, we should review the changes there too.

Reply:
There is the RP  https://github.com/juicedata/go-fuse/pull/2 .